ament_cmake_python/cmake/ament_python_install_package.cmake

```diff
@@ -134,19 +134,18 @@ setup(
 
   # Install as flat Python .egg to mimic https://github.com/colcon/colcon-core
   # handling of pure Python packages.
-  file(RELATIVE_PATH install_dir "${build_dir}" "${CMAKE_INSTALL_PREFIX}")
 
   # NOTE(hidmic): Allow setup.py install to build, as there is no way to
   # determine the Python package's source dependencies for proper build
   # invalidation.
   install(CODE
     "message(STATUS \"Installing: ${package_name} as flat Python egg \"
-                    \"to ${CMAKE_INSTALL_PREFIX}/${PYTHON_INSTALL_DIR}\")
+                    \"to \$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/${PYTHON_INSTALL_DIR}\")
```

Comment:
DESTDIR + CMAKE_INSTALL_PREFIX + PYTHON_INSTALL_DIR seems to be the same order used in catkin, so this looks fine to me

https://github.com/ros/catkin/blob/203a57ad70430ae165ebe014738a37c3af3e9166/cmake/templates/python_distutils_install.sh.in#L19